synapse/models/infotech.py

```diff
@@ -934,6 +956,103 @@ def getModelDefs(self):
                         'doc': 'An array of ChildOf CWE Relationships.'
                     }),
                 )),
+
+                ('it:sec:metrics', {}, (
+
+                    ('org', ('ou:org', {}), {
+                        'doc': 'The organization whose security program is being measured.'}),
+
+                    ('org:name', ('ou:name', {}), {
+                        'doc': 'The organization name. Used for entity resolution.'}),
+
+                    ('org:fqdn', ('inet:fqdn', {}), {
+                        'doc': 'The organization FQDN. Used for entity resolution.'}),
+
+                    ('window', ('ival', {}), {
+                        'doc': 'The time window used to compute the metrics.'}),
+
+                    ('alerts:meantime:triage', ('duration', {}), {
+                        'doc': 'The mean time to triage for alerts generated within the time window.'}),
+
+                    ('alerts:count', ('int', {}), {
+                        'doc': 'The total number of alerts generated within the time window.'}),
+
+                    ('alerts:falsepos', ('int', {}), {
+                        'doc': 'The number of alerts generated within the time window that were determined to be false positives.'}),
+
+                    ('assets:hosts', ('int', {}), {
+                        'doc': 'The total number of hosts within scope for the information security program.'}),
+
+                    ('assets:users', ('int', {}), {
+                        'doc': 'The total number of users within scope for the information security program.'}),
+
+                    ('assets:vulns:count', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities being tracked at the end of the window.'}),
+
+                    ('assets:vulns:preexisting', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities being tracked at the beginning of the window.'}),
+
+                    ('assets:vulns:discovered', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities discovered during the window.'}),
+
+                    ('assets:vulns:mitigated', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities mitigated during the window.'}),
+
+                    ('assets:vulns:meantime:mitigate', ('duration', {}), {
+                        'doc': 'The mean time to mitigate for vulnerable assets mitigated during the time window.'}),
+
+                )),
+
+                ('it:sec:vuln:scan', {}, (
+
+                    ('time', ('time', {}), {
```

Comment:
do we want to capture the scan completion time?

Reply:
gonna table this one for now. we can discuss more.